### PR TITLE
PEP8 Fixes and String Formatting Enhancement

### PR DESCRIPTION
**Pull request by vkmrishad**
### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

### Short description of what this PR does:
- PEP 8 Fixes.
- String Formatting Enhancement.

If you have questions, please send an email to [SendGrid](mailto:dx@sendgrid.com), or file a GitHub Issue in this repository.

### CODE REVIEW

**Comment by vkmrishad** [comment]
hi @misterdorm,
Can you please check this PR

**Comment by misterdorm** [comment]
Replaces #673

**Comment by vkmrishad** [comment]
@misterdorm,
This will replace #673

**Comment by vkmrishad** [comment]
@misterdorm
Please merge this PR

**Comment by vkmrishad** [comment]
@42B @misterdorm 
Can you please merge this PR.

File: test/test_sendgrid.py [hunk 1]
```diff
@@ -144,8 +144,11 @@ def test_hello_world(self):
         content = Content(
             "text/plain", "and easy to do anywhere, even with Python")
         mail = Mail(from_email, subject, to_email, content)
-        self.assertTrue(mail.get() == {'content': [{'type': 'text/plain', 'value': 'and easy to do anywhere, even with Python'}], 'personalizations': [
-                        {'to': [{'email': 'test@example.com'}]}], 'from': {'email': 'test@example.com'}, 'subject': 'Sending with SendGrid is Fun'})
+        self.assertTrue(mail.get() == {
```
Comment (42B) [inline test/test_sendgrid.py]:
```python
self.assertEqual(mail.get, {...})  # ?
```

Comment (vkmrishad) [reply test/test_sendgrid.py]: @42B 
I have updated file with master branch. This file was changed when merged with others PR.

File: test/test_sendgrid.py [hunk 2]
```diff
@@ -144,11 +145,11 @@ def test_hello_world(self):
...
-        self.assertTrue(mail.get() == {
-            'content': [{'type': 'text/plain', 'value': 'and easy to do anywhere, even with Python'}],
-            'personalizations': [{'to': [{'email': 'test@example.com'}]}], 'from': {'email': 'test@example.com'},
-            'subject': 'Sending with SendGrid is Fun'
-        })
+        self.assertTrue(
+            mail.get() == {'content': [{'type': 'text/plain', 'value': 'and easy to do anywhere, even with Python'}],
+                           'personalizations': [
+                               {'to': [{'email': 'test@example.com'}]}], 'from': {'email': 'test@example.com'},
+                           'subject': 'Sending with SendGrid is Fun'})
```
Comment (42B) [inline test/test_sendgrid.py]: It still seems like this should be changed from `self.assertTrue` to `self.assertEqual`? Maybe something like this?

```python
self.assertEqual(mail.get(), {
    'content': [{
        'type': 'text/plain',
        'value': 'and easy to do anywhere, even with Python'
    }],
    'personalizations': [{
        'to': [{'email': 'test@example.com'}]
    }],
    'from': {'email': 'test@example.com'},
    'subject': 'Sending with SendGrid is Fun'
})

```

Comment (vkmrishad) [reply test/test_sendgrid.py]: @42B 
You can create a new issue for this. I used the same code from master.

**Comment by thinkingserious** [comment]
Hello @vkmrishad,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by vkmrishad** [comment]
@thinkingserious @misterdorm @42B 
Thank you

**Pull request merged**